cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/PullSelector.java

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
I don't think this class is really necessary because all it does is wrap a `String`. I think you should remove it and just use `String` directly where this is currently used.